parser/src/main.rs

```diff
@@ -57,7 +58,20 @@ impl Prompt {
         let (lexer_in, lexer_out): (Sender<Token>, Receiver<Token>) = mpsc::channel(32);
         let (normalizer_in, normalizer_out): (Sender<Token>, Receiver<Token>) = mpsc::channel(32);
 
-        let context = MathContext::new();
+        let mut context = MathContext::new();
+        context.variables.insert(
+            MathIdentifier {
+                tokens: vec![Token::Identifier("x".to_string())],
+            },
+            3.0,
+        );
+        context.variables.insert(
+            MathIdentifier {
+                tokens: vec![Token::Identifier("y".to_string())],
+            },
+            7.0,
+        );
+
```

Comment:
Pusha bara inte detta, vill inte upstream kod ska innehålla saker för att testa. Om du har dependencies på detta i någon test rek att du skriver om funktionens definition så att det kan ta in en MathContext.

Reply:
I agree! Resolved